src/appkit.cs

```diff
@@ -589,8 +589,13 @@ interface NSApplication : NSAccessibilityElementProtocol, NSUserInterfaceValidat
 		[Availability (Deprecated = Platform.Mac_10_12, Message = "Use EnumerateWindows instead.")]
 		NSWindow MakeWindowsPerform (Selector aSelector, bool inOrder);
 	
+		[Deprecated (PlatformName.MacOSX, 10, 7, message : "Remove usage or use 'DangerousWindows' instead.")]
```

Comment:
It should be using `[Obsolete]` so the compiler will warn on usage - it won't for `[Deprecated]`. Also I think it _automagically_ removes it (or is it a strikethrough?) from IntelliSense.

`[Deprecated]` is meant to match Apple deprecations (not our own changes). At some point we might want intro/xtro to ensure this match, and not following the guidelines will bite us back

Reply:
I was concerned about doing that as some customers have warn as errors enabled, but sure. I'm happy to make it obsolete.

Reply:
I'm guessing most people using `warnaserror` would be happy to spot this potential issue in their app :)